examples/eric-fail.rb

```diff
@@ -1,32 +0,0 @@
-def test
-  connection = Fog::Compute.new(:provider => "Google")
```

Comment:
I can't remember the original bug, but there was some weird way that @erjohnso was using Fog in another library, which caused this script to blow up. It was interesting at the time, now it's just funny :p